api/src/components/link/service.ts

```diff
@@ -1,4 +1,16 @@
 import * as client from 'lib/client';
+import * as I from 'interface';
+
+export const getPublicationLinks = async (filter: I.GetLinksQueryParams) => {
+    console.log(filter.publicationID)
```

Comment:
probably not useful for debugging as stands so either remove or make a clearer message.